sass/_sponsors.scss

```diff
@@ -0,0 +1,31 @@
+#sponsorSection {
+  text-align: center;
+  display: block;
+  padding: 20px;
+}
+
+#sponsorSection h2 {
+  font-family: SaxMono;
+  font-size: 62px;
+}
+
+.sponsorLogo img {
+ padding: 10px;
+}
+
+#sponsorSection p{
+  font-family: lato;
+  font-size: 24px;
+  padding: 20px;
+}
+
+.sponsorLogo {
+  margin-bottom: 0px;
+  padding-bottom: 10px;
+  padding-top:20px;
+  display: block;
+}
+
+.first-row{
+  padding-bottom: 30px;
+}
```

Comment:
All files should end with an extra return so that Git can use appropriate line endings.